### PR TITLE
Launch IntersectionObserverPolyfill experiments

### PR DESCRIPTION
**Pull request by dvoytenko**
Partial for: #27807

### CODE REVIEW

File: build-system/global-configs/experiments-const.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "INTERSECTION_OBSERVER_POLYFILL": false,
+  "INTERSECTION_OBSERVER_POLYFILL": true,
```
Comment (zhouyx) [inline build-system/global-configs/experiments-const.json:2]: wait did we accidentally ship the polyfill to inabox instead?

Comment (zhouyx) [reply build-system/global-configs/experiments-const.json:2]: hmmmm saw your comment in #28434

Comment (dvoytenko) [reply build-system/global-configs/experiments-const.json:2]: I was told that it it didn't launch to inabox because config.json took precedence.

Comment (zhouyx) [reply build-system/global-configs/experiments-const.json:2]: Got it. Thanks
https://github.com/ampproject/amphtml/blob/b1cdabd46601c687549492c86b00afbd1ed3c711/build-system/babel-config/helpers.js#L79-L89

**Comment by zhouyx** [comment]
🎉

**Pull request merged**